src/assets/environment.json

```diff
@@ -3,7 +3,9 @@
     "environment": "development",
     "apiRoot": "https://api.staging.ecosounds.org",
     "clientOrigin": "http://development.ecosounds.org:4200",
-    "clientDir": "/"
+    "clientDir": "/",
+    "oldClientOrigin": "https://api.staging.ecosounds.org",
+    "oldClientDir": "/listen_to/index.html#"
```

Comment:
`index.html#` is technically not a directory. Could call it `base` instead of `dir`, could also just put index.html# in the code